metrics-facade-spring/metrics-facade-prometheus-spring-boot-autoconfigure/src/main/java/com/ringcentral/platform/metrics/spring/prometheus/MfPrometheusAutoConfiguration.java

```diff
@@ -71,15 +72,16 @@ public PrometheusMetricsExporter prometheusMetricsExporter(
 
         MfPrometheusConfig config = configBuilder.build();
 
-        InstanceSamplesProvider<? extends PrometheusSample, ? extends PrometheusInstanceSample> instanceSamplesProvider =
+        final var instanceSamplesProvider =
```

Comment:
It's pretty difficult to understand from the context type of created instance.

I would suggest to declare in both places types explicitly.

Reply:
ok, will roll it back, thank you!